src/zcl/definition/cluster.ts

```diff
@@ -4113,6 +4113,12 @@ const Cluster: {
         commands: {},
         commandsResponse: {},
     },
+    manuSpecificTuya_1: {
+        ID: 0xE000,
+        attributes: {},
+        commands: {},
+        commandsResponse: {},
```

Comment:
Since there are no attributes/commands/commandReponses how is this cluster used?